schema.graphql

```diff
@@ -254,47 +266,53 @@ type Scope @jsonField {
 
 type ClaimScope @entity {
   id: ID!
-  targetDid: String! @index
+  target: String! @index(unique: false)
```

Comment:
This one could have been left as targetDid, since its not a reference type.

OK to leave if you think `target` is better though.